signer/src/transaction_coordinator.rs

```diff
@@ -210,14 +218,20 @@ where
 
         let transaction_package = pending_requests.construct_transactions()?;
 
-        for transaction in transaction_package {
+        for mut transaction in transaction_package {
             self.sign_and_broadcast(
                 bitcoin_chain_tip,
                 aggregate_key,
                 signer_public_keys,
-                transaction,
+                &mut transaction,
             )
             .await?;
+
+            // TODO: need to add a retry (somewhere else?) if this fails?
```

Comment:
This should retry on its own if it's using an `ApiFallbackClient` (which it does in `main.rs`)

Reply:
I was thinking about the case of Emily being offline or unreachable: since we already submitted the tx we would never mark it as accepted. That's why I was thinking of adding the retry in some other place, looking for inconsistencies.

Reply:
Ahh okay :+1: